test/runtests.jl

```diff
@@ -1130,3 +1130,7 @@ end
         end
     end
 end
+
+@testset "Docstrings" begin
+    @test Docs.undocumented_names(Tar) == []
+end
```

Comment:
You need to run this test only in recent Julia versions:
```suggestion
if isdefined(Docs, :undocumented_names) # new in Julia 1.11
    @testset "Docstrings" begin
        @test Docs.undocumented_names(Tar) == []
    end
end
```

Reply:
Maybe you can adapt the second line of https://github.com/JuliaLang/julia/issues/52725#issue-2064789767, where I copied the test from?